src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -712,4 +712,19 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="AddClientCertCommandDescription" xml:space="preserve">
     <value>Adds a client certificate configuration that matches the given package source name.</value>
   </data>
-</root>
+  <data name="AddPkg_PackagePrerelease" xml:space="preserve">
+    <value>Allows prerelease packages to be installed.</value>
+  </data>
+  <data name="Error_PrereleaseWhenVersionSpecified" xml:space="preserve">
+    <value>The --prerelease and --version options are not supported in the same command.</value>
+    <comment>Please don't localize "prerelease" and "version"</comment>
+  </data>
+  <data name="Error_NoVersionsAvailable" xml:space="preserve">
+    <value>There are no versions available for the package '{0}'.</value>
+    <comment>{0} - PackageID</comment>
+  </data>
+  <data name="PrereleaseVersionsAvailable" xml:space="preserve">
+    <value>There are no stable version available, {0} is the latest available</value>
```

Comment:
I thought we want to say "best"? :) 

Also make sure you have a `.` at the end of the sentence.